### PR TITLE
Show warnings in UI if query have returned some warnings

### PR DESCRIPTION
**Pull request by svagner**
- show warnings in UI if query have returned some warnings
- improve warning (error) text if query to remote was finished with error

As follow up of MR https://github.com/prometheus/prometheus/pull/5963

### CODE REVIEW

**Comment by svagner** [comment]
@brian-brazil Could you please take a look to this MR?

File: web/ui/static/js/graph/index.js [hunk 1]
```diff
@@ -513,6 +513,10 @@ Prometheus.Graph.prototype.submitQuery = function() {
           return;
         }
 
+        if ("warnings" in json && json.warnings.length > 0) {
+          self.showWarning(json.warnings.join('<br>'));
```
Comment (brian-brazil) [inline web/ui/static/js/graph/index.js]: We should probably prefix this with an indication that it's a remote read issue.

Comment (svagner) [reply web/ui/static/js/graph/index.js]: May be it wpuld be better to add it within error from storage/remote like fmt.Sprintf("remote_read: %s", err). How do you think?

Comment (brian-brazil) [reply web/ui/static/js/graph/index.js]: Makes sense.

Comment (svagner) [reply web/ui/static/js/graph/index.js]: Prefix was added

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**